pkg/kubelet/status/status_manager.go

```diff
@@ -87,8 +86,18 @@ type manager struct {
 	stateFileDirectory string
 }
 
-// PodStatusProvider knows how to provide status for a pod. It's intended to be used by other components
-// that need to introspect status.
+// PodManager is the subset of methods the manager needs to observe the actual state of the kubelet.
+// See pkg/k8s.io/kubernetes/pkg/kubelet/pod.Manager for method godoc.
+type PodManager interface {
```

Comment:
Is there any reason this needs to be exported? Same question in volume manager.

Reply:
on the principle that the interface is being used in a public method.  We have more cases where we export the interface than not, and ALSO you don't see public godoc for private methods, so leaned slightly towards making them all public.  In *general* go style discourages private types in public methods, but it's not hard and fast, and Kube doesn't consistently enforce it.

Reply:
I see, that makes sense - if it's being used in a public method then it makes sense to export the interface as well.